framework/auth/core.py

```diff
@@ -959,6 +985,7 @@ def save(self, *args, **kwargs):
         ret = super(User, self).save(*args, **kwargs)
         if self.SEARCH_UPDATE_FIELDS.intersection(ret) and self.is_confirmed:
             self.update_search()
+            self.update_search_nodes_contributors()
```

Comment:
This may be premature optimization, but since this operation may be expensive, may want to only update if information that would be displayed on the node changes (ie, if social fields are updated, the nodes don't need to know)